src/main/java/de/retest/recheck/cli/utils/ErrorHandler.java

```diff
@@ -0,0 +1,42 @@
+package de.retest.recheck.cli.utils;
+
+import java.io.File;
+import java.io.IOException;
+import java.nio.file.NoSuchFileException;
+
+import com.esotericsoftware.kryo.KryoException;
+
+import de.retest.recheck.Properties;
+import de.retest.recheck.cli.TestReportFormatException;
+import lombok.extern.slf4j.Slf4j;
+
+@Slf4j
+public class ErrorHandler {
```

Comment:
@beatngu13 Do we want some tests for this class or rely on the integration tests?